packages/wallets/src/features/cfd/screens/TradingPlatformStatus/TradingPlatformStatus.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { WalletButton, WalletText } from '../../../../components/Base';
+import { useModal } from '../../../../components/ModalProvider';
+import useDevice from '../../../../hooks/useDevice';
+import './TradingPlatformStatus.scss';
+
+type TradingPlatformStatusModalProps = {
+    isServerMaintenance: boolean;
+};
+
+const TradingPlatformStatus: React.FC<TradingPlatformStatusModalProps> = ({ isServerMaintenance }) => {
+    const { hide } = useModal();
+    const { isMobile } = useDevice();
+    const { t } = useTranslation();
+
+    const title = isServerMaintenance ? 'Server Maintenance' : 'Account Unavailable';
+    const content = isServerMaintenance
+        ? 'We’re currently performing server maintenance. Service maybe affected.'
+        : 'The server is temporarily unavailable for this account. We’re working to resolve this.';
```

Comment:
The texts should be wrapped in localize directly i.e:
```suggestion
const { localize } = useTranslations()
        ? localize('We’re currently performing server maintenance. Service maybe affected.')
        : localize('The server is temporarily unavailable for this account. We’re working to resolve this.');
```

as dynamic values passed will not get uploaded into crowdin